website/blog/2025-11-24-aks-automatic-managed-system-node-pools/index.md

```diff
@@ -0,0 +1,172 @@
+| --- | --- | --- |
+| **Provisioning** | You create the pool, select VM SKUs, set node count, and configure OS disk size | AKS provisions and sizes the pool for you automatically |
+| **Capacity planning** | You estimate headroom for system components like CoreDNS, konnectivity, metrics-server, and any add-ons; scale manually or configure cluster autoscaler with min/max counts | AKS right-sizes capacity for platform components and scales automatically when add-ons need more room |
+| **Cost** | System nodes are billed as standard VMs to your subscription; you pay for system pool capacity | System nodes do not run on your subscription |
+
+![Choose your operating model between AKS Standard and AKS Automatic](aks-standard-automatic.png)
+
+## Components running on managed system node pools
+
+AKS manages the following platform components on the managed system node pool. You don't need to provision capacity for these services:
+
+| Component | Description |
+| --- | --- |
+| [Azure Monitor](https://learn.microsoft.com/azure/aks/monitor-aks) | Collects logs, metrics, and Kubernetes state for observability dashboards and alerts |
+| [CoreDNS](https://learn.microsoft.com/azure/aks/coredns-custom) | Provides cluster DNS resolution for service discovery |
+| [Eraser](https://learn.microsoft.com/azure/aks/image-cleaner) | Removes unused and vulnerable container images from nodes |
+| [KEDA](https://learn.microsoft.com/azure/aks/keda-about) | Scales workloads based on event-driven metrics such as queue length or HTTP traffic |
+| Konnectivity | Maintains secure connectivity between the control plane and nodes |
+| [Metrics Server](https://learn.microsoft.com/azure/aks/monitor-aks-reference) | Exposes resource metrics for Horizontal Pod Autoscaler and kubectl top |
+| [VPA](https://learn.microsoft.com/azure/aks/vertical-pod-autoscaler) | Recommends and applies optimal CPU and memory requests for pods |
+| [Workload Identity](https://learn.microsoft.com/azure/aks/workload-identity-overview) | Enables pods to authenticate to Azure services using Microsoft Entra ID |
+
+Other add-ons and extensions run on `aks-system-surge` nodes, with scaling handled by [Node Auto-Provisioning (NAP)](https://learn.microsoft.com/azure/aks/node-auto-provisioning). `DaemonSets` run on both managed system node pools and nodes in your subscription.
+
+## Guardrails for security and reliability
+
+Security misconfigurations are a leading cause of container breaches. AKS Automatic addresses this by enforcing [Deployment Safeguards](https://learn.microsoft.com/azure/aks/deployment-safeguards) that validate every workload against the [Kubernetes Pod Security Standards](https://kubernetes.io/docs/concepts/security/pod-security-standards/) before it reaches your cluster. Baseline policies block dangerous privilege escalations while restricted policies enforce maximum hardening. Compliance flows into Azure Policy dashboards automatically.

```

Comment:
what are our baseline policies in aks automatic - if thats an exhaustive list pls list or link to that?